lib/mongodb.js

```diff
@@ -25,7 +26,7 @@ exports.initialize = function initializeSchema(schema, callback) {
 
 var MongoObjectID = function ObjectID(id) {
```

Comment:
We have a function and a variable both named `ObjectId`; can we rename or even delete the variable and use `bson.ObjectId` in the few places where it's needed?